src/components/DetailsCard/renderSocialMediaIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { ReactElement } from 'react';
+import { GitHub, Facebook, Twitter, Instagram } from 'react-feather';
+
+export const renderSocialMediaIcon = (socialMediaName: string, className: string): ReactElement => {
```

Comment:
Use an object with keys `facebook`, `github`, `twitter` and `instagram` instead of a  function.

Reply:
Could you please elaborate on that, I do not understand/see the implementation that you suggest 😄

Reply:
```
const socialMediaIcons = {
    facebook: Facebook,
    github: GitHub,
    twitter: Twitter,
    instagram: Instagram,
};

export const renderSocialMediaIcon = (socialMediaName: SocialMediaName, className: string): ReactElement => {
    const Icon = socialMediaIcons[socialMediaName];
    
    return <Icon className={className} />
};
```

Reply:
When I said "instead of a function" I meant you can just export the object and use the line `const Icon = socialMediaIcons[socialMediaName];` in places you need it. But you're right, it would be better to just simplify the function to the example above and still use it in all places.

Reply:
Changed ✅